### PR TITLE
itests: add asset force close itest

### PR DESCRIPTION
**Pull request by Roasbeef**
In this PR, we add the `AuxSweeper`, update the lnd+tapd dependancies, and also add an initial force close itest.

### CODE REVIEW

File: itest/litd_custom_channels_test.go [hunk 1]
```diff
@@ -618,6 +630,343 @@ func testCustomChannels(_ context.Context, net *NetworkHarness,
...
+			t.t, charlie, dave, keySendAmount, assetID,
+			fn.Some(btcAmt),
+		)
+	}
+
+	logBalance(t.t, nodes, assetID, "after keysend")
+
+	// With the payments sent, we'll now go on chain with a force close
+	// from Carol.
+	t.Logf("Force closing channel...")
```
Comment (GeorgeTsagk) [inline itest/litd_custom_channels_test.go:801]: nit: could also make some payments from dave to charlie for extra coverage

**Review by GeorgeTsagk — COMMENTED** [review]
something in the dependencies is off
itest doesn't compile

**Comment by guggero** [comment]
Pushed some updates to make things compile.

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Review by GeorgeTsagk — APPROVED** [review]
ty @guggero for the fixes
LGTM ✔️ 
pending some nits from above, but non-blocking

**Comment by Roasbeef** [comment]
Have a breach follow up coming in another PR.

**Pull request merged**